dome/comparison.md

```diff
@@ -0,0 +1,276 @@
+Gaussian noise | ✓ | ✓
+Custom sensors | ✓ | [Issue](https://github.com/ignitionrobotics/ign-sensors/issues/9)
+
+## SDF Features
+
+Feature | Gazebo-classic | Ignition Gazebo
+-- | -- | --
+SDF frame semantics |✓| ✓
+Load models from local files | ✓ | [✕](https://github.com/ignitionrobotics/ign-gazebo/issues/123)
+Closed kinematic chains | ✓  | [✕](https://github.com/ignitionrobotics/ign-physics/issues/25)
+Nested models | ✓ | Partial support
+Populations | ✓ | [Issue](https://github.com/ignitionrobotics/ign-gazebo/issues/240)
+Actors | ✓ | ✓
+Markers | ✓ | ✓
+Heightmaps | ✓ | [Issue](https://github.com/ignitionrobotics/ign-gazebo/issues/237)
+DEM (Digital Elevation Models) | ✓ | [Issue](https://github.com/ignitionrobotics/ign-gazebo/issues/235)
+Polylines | ✓ | [Issue](https://github.com/ignitionrobotics/ign-gazebo/issues/186)
+World plugins | ✓ | ✓ Now called System plugin
+Model plugins | ✓ | ✓ Now called System plugin
+Sensor plugins | ✓ | ✓ Now called System plugin
+Visual plugins | ✓ | [Issue](https://github.com/ignitionrobotics/ign-gazebo/issues/265)
+GUI plugins | ✓ | ✓ Ignition GUI plugins and Gazebo GUI systems
+System plugins | ✓ | ✓ Through Ignition Launch
+
+## Plugins
+
+### Model plugins
+
+Plugin | Gazebo-classic | Ignition Gazebo
+-- | -- | --
+ActorPlugin | ✓ | ✕ See [FollowActor](https://github.com/ignitionrobotics/ign-gazebo/blob/master/src/systems/follow_actor/FollowActor.hh) for a demo of Actor APIs
+ActuatorPlugin | ✓ |
+ArduCopterPlugin | ✓ |
+AttachLightPlugin | ✓ | ✕ Does not apply, use SDF
+Breadcrumbs | ✕ | ✓
+BuoyancyPlugin | ✓ | [Issue](https://github.com/ignitionrobotics/ign-gazebo/issues/159)
+CartDemoPlugin | ✓ | ✕
+CessnaPlugin | ✓ |
+DiffDrivePlugin | ✓ | ✓
+ElevatorPlugin | ✓ |
+FlashLightPlugin | ✓ |
+FollowerPlugin | ✓ |
+GimbalSmall2dPlugin | ✓ |
+GravityCompensationPlugin | ✓ |
+HarnessPlugin | ✓ |
+HydraDemoPlugin | ✓ |
+InitialVelocityPlugin | ✓ | [Issue](https://github.com/ignitionrobotics/ign-gazebo/issues/50)
+JointControlPlugin | ✓ (force / pos / vel, from SDF) | ✓ (vel, from msg)
+JointStatePublisher | ✕ | ✓
+JointTrajectoryPlugin | ✓ |
+KeysToCmdVelPlugin | ✓ | Use `ignition::gui::KeyPublisher` with `ignition::gazebo::systems::TriggeredPublisher`
+KeysToJointsPlugin | ✓ | Use `ignition::gui::KeyPublisher` with `ignition::gazebo::systems::TriggeredPublisher`
+LedPlugin | ✓ |
+LiftDragPlugin | ✓ | ✓
+LinearBatteryConsumerPlugin | ✓ | ✓
+LinearBatteryPlugin | ✓ | ✓
+LinkPlot3DPlugin | ✓ | [Issue](https://github.com/ignitionrobotics/ign-gazebo/issues/231)
+MudPlugin | ✓ |
+MulticopterMotorModel | ✕ | ✓
+PlaneDemoPlugin | ✓ |
+PosePublisher | ✕ | ✓
+RandomVelocityPlugin | ✓ |
+RegionEventBoxPlugin | ✓ |
+SimpleTrackedVehiclePlugin | ✓ |
+SkidSteerDrivePlugin | ✓ | ✓
+SphereAtlasDemoPlugin | ✓ | ✕
+TouchPlugin | ✓ | ✓
+TrackedVehiclePlugin | ✓ |
+VariableGearboxPlugin | ✓ |
+VehiclePlugin | ✓ |
+WheelSlipPlugin | ✓ | [pull request](https://github.com/ignitionrobotics/ign-gazebo/pull/134)
```

Comment:
This PR has actually been merged, so I'm not sure if we want to update the chart to reflect this or not.

Reply:
Thanks, updated: e0d9b86